datafusion/proto/proto/datafusion.proto

```diff
@@ -507,17 +507,6 @@ message ScalarUDFExprNode {
 
 enum BuiltInWindowFunction {
   UNSPECIFIED = 0; // https://protobuf.dev/programming-guides/dos-donts/#unspecified-enum
-  // ROW_NUMBER = 0;
-  //  RANK = 1;
-  //  DENSE_RANK = 2;
-  //  PERCENT_RANK = 3;
-  //  CUME_DIST = 4;
-  //  NTILE = 5;
-  // LAG = 6;
-  // LEAD = 7;
-  FIRST_VALUE = 8;
-  LAST_VALUE = 9;
-  NTH_VALUE = 10;
```

Comment:
Instead of removing everything just comment out the enums. After the change you will need to run `regen.sh` again.

This too we can tackle in a cleanup PR.

Reply:
commented out the enums